manifests/params.pp

```diff
@@ -29,9 +29,9 @@
     '/var/spool/cups', '/var/spool/squid', '/var/tmp',
   ]
 
-  $_cron_min  = fqdn_rand(60)
-  $_cron_hour = fqdn_rand(24)
-  $_cron_day  = fqdn_rand(7)
+  $_cron_min  = fqdn_rand(60,'min')
```

Comment:
Adding min, hour, day won't have any meaningful impact.  If other fqdn_rand calls in other modules use the same settings, the results will all be the same.  Instead how about change the hour/min/day to simply $module_name?  This will force the the fqdn_rand to be seeded to this specific module.